cmd/validator/flags/flags.go

```diff
@@ -322,6 +322,26 @@ var (
 		Usage: "Enables more verbose logging for counting down to duty",
 		Value: false,
 	}
+
+	// ValidatorsProposerConfigDirFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigDirFlag = &cli.StringFlag{
+		Name:  "validators-proposer-config-dir",
+		Usage: "remote local file path to load proposer configuration from",
+		Value: "",
+	}
+	// ValidatorsProposerConfigFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigURLFlag = &cli.StringFlag{
+		Name:  "validators-proposer-config-url",
+		Usage: "remote URL file path to load proposer configuration from",
+		Value: "",
+	}
+
+	// SuggestedFeeRecipientFlag defines the address of the fee recipient.
+	SuggestedFeeRecipientFlag = &cli.StringFlag{
+		Name:  "suggested-fee-recipient",
+		Usage: "The suggested eth1 address to receive fees from ALL validator proposers",
+		Value: "0x0",
```

Comment:
Should we make this value equal to the 20 byte long zero address?

Reply:
replaced with eth burn address